src/cljs/rems/user_settings.cljs

```diff
@@ -80,10 +80,20 @@
                    (when first-time? ; to avoid an infinite loop if the settings fail to save (e.g. unsupported language in cookies)
                      [[:check-if-should-save-language!]]))})))
 
+(rf/reg-event-fx
+ :loaded-user-settings-fail
+ (fn [{:keys [db]} [_ user-settings-error]]
+   (let [logged-in? (:user (:identity db))]
+     (if-not logged-in?
+       {:db db
+        :dispatch (when-not (= 401 (:status user-settings-error))
```

Comment:
could combine this when with the above if

Reply:
addressed